pynb.py

```diff
@@ -17,6 +18,10 @@
 import paramiko
 SSH_PORT = 22
 DEFAULT_PORT = 8888
+DEFAULT_CPU = 8
+DEFAULT_MEM = '32G'
+DEFAULT_PARTITION = 'ei-medium' 
+DEFAULT_ENV= '/usr/users/EI_ga012/%s/something1234' % os.environ['USER']
```

Comment:
I wouldn't have the default environment here, that should be configured either by the package manager or the user's shell setup.